### PR TITLE
Feature/order section skeleton

### PR DESCRIPTION
**Pull request by Julian0701**
### DEVELOP
- [x] open_sub_tab.tsx
- [x] history_sub_tab.tsx

### CHECK LIST
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 1
    - react-loading-skeleton: 3.2.1
- new Class / Component: 0
- new loop: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/open_position_item/open_position_item.tsx [hunk 1]
```diff
@@ -49,6 +42,16 @@ const OpenPositionItem = ({openCfdDetails, ...otherProps}: IOpenPositionItemProp
   const openItemClickHandler = () => {
     dataUpdateFormModalHandler(openCfdDetails);
     visibleUpdateFormModalHandler();
+
+    toast({
```
Comment (Julian0701) [inline src/components/open_position_item/open_position_item.tsx:46]: 移除

File: src/components/withdrawal_modal/withdrawal_modal.tsx [hunk 2]
```diff
@@ -98,7 +100,11 @@ const WithdrawalModal = ({
       const result = await userCtx.withdraw(withdrawOrder);
 
       // TODO: for debug
-      globalCtx.toast({message: 'withdraw result: ' + JSON.stringify(result), type: 'info'});
+      globalCtx.toast({
```
Comment (Julian0701) [inline src/components/withdrawal_modal/withdrawal_modal.tsx:103]: 移除

File: src/contexts/global_context.tsx [hunk 3]
```diff
@@ -655,14 +657,16 @@ export const GlobalProvider = ({children}: IGlobalProvider) => {
       ? () => {
           setDataLoadingModal(modalReOpenData),
             setVisibleLoadingModal(true),
-            toastify.dismiss(type + message + toastId);
+            toastify.dismiss(isToastId);
```
Comment (Julian0701) [inline src/contexts/global_context.tsx:660]: 實驗: 如果 dismiss 不存在的 id 會不會出錯

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**